.github/workflows/test-linux.yml

```diff
@@ -0,0 +1,52 @@
+name: "Test Ark - Linux"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      ssh:
+        description: 'Set up an SSH session before running `cargo test`?'
+        type: boolean
+        required: true
+        default: false
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    name: "Rust: ${{ matrix.config.rust }}, R: ${{ matrix.config.r }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - { rust: 'stable',  r: 'release' }
+          # Oldest supported R version
+          - { rust: 'stable',  r: '4.2' }
+          # Nightly rust
+          - { rust: 'nightly', r: 'release' }
+    timeout-minutes: 30
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: dtolnay/rust-toolchain@nightly
+        if: matrix.config.rust == 'nightly'
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: ${{ matrix.config.r }}
+          use-public-rspm: true
```

Comment:
Does this mean we can install binary packages from p3m?

By the way there's a bunch of tests skipped because data.table and tibble are not installed on CI. We could now easily fix that right?

E.g. using https://github.com/r-lib/actions/tree/v2/setup-r-dependencies with `extra-packages` or something (hopefully it still works without a `DESCPRIPTION` file but we could add one if needed).

Reply:
https://github.com/posit-dev/ark/pull/545/commits/b016ea97557734893018080ae6a3ac1a175cc10f